pkg/resource/key/key_rotation_hooks.go

```diff
@@ -0,0 +1,82 @@
+package key
```

Comment:
We need to prepend the license boilerplate to this file

Reply:
Resolved